onnxruntime/test/testdata/transform/fusion/bias_dropout_residual_gen.py

```diff
@@ -0,0 +1,114 @@
+import onnx
+from onnx import helper
+from onnx import TensorProto, OperatorSetIdProto
+
+# inputs/outputs
+A = helper.make_tensor_value_info('A', TensorProto.FLOAT, ['unk_1', 'unk_2', 3072])
+B = helper.make_tensor_value_info('B', TensorProto.FLOAT, [3072])
+R = helper.make_tensor_value_info('R', TensorProto.FLOAT, ['unk_1', 'unk_2', 3072])
+C = helper.make_tensor_value_info('C', TensorProto.FLOAT, ['unk_1', 'unk_2', 3072])
+mask = helper.make_tensor_value_info('mask', TensorProto.BOOL, ['unk_1', 'unk_2', 3072])
+
+# initializers
+ratio = helper.make_tensor('ratio_const', TensorProto.FLOAT, [], [0.8])
+training_mode = helper.make_tensor('training_mode', TensorProto.BOOL, [], [1])
+
+opsets = []
+onnxdomain = OperatorSetIdProto()
+onnxdomain.version = 12
+onnxdomain.domain = "" # The empty string ("") or absence of this field implies the operator set that is defined as part of the ONNX specification.
+opsets.append(onnxdomain)
+
+kwargs={}
+kwargs['opset_imports'] = opsets
+
+# Create the model (ModelProto)
+bias = helper.make_node("Add", ["A", "B"], ["add0_out"], "add0")
+dropout_12 = helper.make_node("Dropout", ["add0_out", "ratio_const", "training_mode"], ["C", "mask"], "dropout0")
```

Comment:
should we add "seed" attribute as well?

Reply:
I think it can skipped... 

---
In reply to: [441091186](https://github.com/microsoft/onnxruntime/pull/4167#discussion_r441091186) [](ancestors = 441091186)